logs/Scholarlify-BPP-Financial Support/on_search-response.json

```diff
@@ -0,0 +1,22 @@
+{
+  "context": {
+    "domain": "onest:financial-support",
+    "action": "on_search",
+    "version": "1.1.0",
+    "bpp_id": "onest-dev.scholarlify.com",
+    "bpp_uri": "https://onest-dev.scholarlify.com",
+    "country": "IND",
+    "city": "*",
+    "bap_id": "fs-ps-bap-network.onest.network",
+    "bap_uri": "https://fs-ps-bap-network.onest.network",
+    "transaction_id": "41bcfd01-da1c-4c4d-b39e-4418779aea80",
+    "message_id": "ac1756d1-0f44-4321-8abc-3509941ad921",
+    "ttl": "PT10M",
+    "timestamp": "2024-05-28T08:41:54.199Z"
+  },
+  "message": {
+    "ack": {
+      "status": "ACK"
```

Comment:
Ack response is not required. These JSON files can be removed from this folder

Reply:
ok removed the response files